### DISCUSSION TITLE
iOS版是不能用吗？

### DISCUSSION THREAD

**Discussion by AAABC092** [getlantern/lantern-forum-cn#1022] [Q&A]
听说账号可以绑定iOS系统了，下载了最新版本，但是绑定后从来没连上过网络，而且PC端今年也超级不稳定，经常断连。用了好多年了，今年真是最难用的一年，希望能快点改进一下。
![屏幕截图 2024-09-26 191301](https://github.com/user-attachments/assets/8b052eaa-771b-4fef-8d34-0d8bbcf67337)

**Comment by MaxMKMax** [comment]
@AAABC092 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
